server/routes/api/chart.py

```diff
@@ -110,7 +110,8 @@ def geojson(dcid):
     if not geos:
         return Response(json.dumps({}), 200, mimetype='application/json')
 
-    names_by_geo = place_api.get_display_name('^'.join(geos))
+    locale = request.args.get('hl', default="en")
```

Comment:
And we should update the cache line above on line 104 to:
@cache.cached(timeout=<>, query_string=True)

otherwise the cache will hold whichever language was requested first.